scala-list.cfg

```diff
@@ -0,0 +1,6 @@
+# scala versions to build images for
+2.11.8
```

Comment:
Is there any reason we want the various versions instead of just the latest for this first build